ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -0,0 +1,71 @@
+package ipuplugin
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	pb2 "github.com/openshift/dpu-operator/tree/main/dpu-api/gen"
+	"google.golang.org/protobuf/types/known/emptypb"
+	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+)
+
+type DevicePluginService struct {
+	pb2.UnimplementedDeviceServiceServer
+}
+
+var (
+	exclude     = []string{"enp0s1f0", "enp0s1f0d2"}
```

Comment:
possibly enp0s1f0d1 should also be excluded as it may be used for communication with a physical port

Reply:
added